src/main/java/org/folio/services/ftp/FtpUploadService.java

```diff
@@ -26,14 +27,17 @@
 public class FtpUploadService implements FileExchangeService {
 
   private static final Logger logger = LogManager.getLogger(FtpUploadService.class);
-
   private static final String DEFAULT_WORKING_DIR = "/files/invoices";
+  public static final String URL_NOT_FOUND_FOR_FTP = "URL for FTP upload was not found";
```

Comment:
need to standartize to the same format, here is URL and in the similar error URI is used

Reply:
Thanks, changed